internal/service/jobs_tf/model.go

```diff
@@ -22,13 +22,36 @@ type BaseJob struct {
 	// The creator user name. This field won’t be included in the response if
 	// the user has already been deleted.
 	CreatorUserName types.String `tfsdk:"creator_user_name" tf:"optional"`
+	// The id of the budget policy used by this job for cost attribution
+	// purposes. This may be set through (in order of precedence): 1. Budget
+	// admins through the account or workspace console 2. Jobs UI in the job
+	// details page and Jobs API using `budget_policy_id` 3. Inferred default
+	// based on accessible budget policies of the run_as identity on job
+	// creation or modification.
+	EffectiveBudgetPolicyId           types.String `tfsdk:"effective_budget_policy_id" tf:"optional"`
+	Effective_EffectiveBudgetPolicyId types.String `tfsdk:"effective_effective_budget_policy_id" tf:"computed,optional"`
```

Comment:
Rare case when the field name itself upstream starts with Effective?

Reply:
Wonderful :)

Reply:
Anyway, dropped underscores.